src/Storage/Extensions/DataTypeExtensions.cs

```diff
@@ -0,0 +1,51 @@
+using System.Threading.Tasks;
```

Comment:
We can slowly add more nullable references to new files. (And old if you have the time.)
```suggestion
#nullable enable
using System.Threading.Tasks;
```
This will highlight the parameter `task`.